<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BootCheck-2015-07-31-debug</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>System BootTimes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_totalTimers">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Boot Interval: </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Check Item:</string>
    </property>
    <widget class="QCheckBox" name="m_checkEnternet">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Enternet</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>times</string>
     </property>
    </widget>
    <widget class="QLabel" name="m_nUSB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="m_checkTF">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>TF SDCARD</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>times</string>
     </property>
    </widget>
    <widget class="QLabel" name="m_nTF">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="m_checkUSB">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>USB</string>
     </property>
    </widget>
    <widget class="QLabel" name="m_nEnternet">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>times</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="m_pushButtonStart">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>StartCheck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pushButtonEnd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>StopCheck</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pushButtonReset">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ResetCheck</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_nInterval">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
